src/index.js

```diff
@@ -1,4 +1,48 @@
-import _ from 'lodash';
 import './style.css';
 
+const taskContainer = document.querySelector('.all-tasks');
+const tasks = [
+  {
+    id: Date.now(),
+    completed: false,
+    description: 'Make Breakfast',
+  },
+  {
+    id: Date.now(),
+    completed: false,
+    description: 'Clean the House',
+  },
+  {
+    id: Date.now(),
+    completed: false,
+    description: 'Report to Work',
+  },
+  {
+    id: Date.now(),
+    completed: false,
+    description: 'join stand up team',
+  },
+];
 
+tasks.forEach((task, e) => {
+  const taskItem = document.createElement('div');
+  taskItem.classList = 'list-item';
+
+  const checkbox = document.createElement('input');
+  checkbox.type = 'checkbox';
+
+  const taskDesc = document.createElement('input');
+  taskDesc.type = 'text';
+  taskDesc.value = `${tasks[e].description}`;
+
+  const ellipses = document.createElement('button');
+  ellipses.innerHTML = '<i class="uil uil-ellipsis-v"></i>';
+
+  taskItem.append(checkbox);
+  taskItem.append(taskDesc);
+  taskItem.append(ellipses);
+
+  taskContainer.append(taskItem);
+  taskDesc.removeAttribute('disabled');
+  taskDesc.focus();
+});
```

Comment:
- [ ] Ouch! this is close but as specified in the project requirements the list should appear in order of the index values for each task. So I'd suggest you replace the `id` property of the task object with an index and render them by the `index` order.
- Project requirement 👇
  ![image](https://github.com/coker2019/To-do-List/assets/104968236/1edac2f5-9343-40a7-ae16-3b176eacd02e)